test/unit/libvips.js

```diff
@@ -12,6 +12,7 @@ const setPlatform = function (platform) {
 
 const restorePlatform = function () {
   setPlatform(originalPlatform);
+  delete process.env.SHARP_IGNORE_GLOBAL_LIBVIPS;
```

Comment:
The platform-agnostic tests don't (currently) call `restorePlatform` so I think this env var won't get deleted. Perhaps the `delete` could move to the new test itself, after the assertion?

Reply:
Ah! Good call 😅  That explains why I saw the `process.env.SHARP_IGNORE_GLOBAL_LIBVIPS` being used 5 times on coveralls. I could explain this to myself.

I'll move the delete to the test itself then unless if I find a cleaner solution :-)